sensors/git_activity_sensor.py

```diff
@@ -109,16 +109,20 @@ def _process_repository(self, name, repository, whitelist):
                                (name, repository))
         assert(isinstance(name, six.text_type))
 
+        last_event_id = self._get_last_id(name=name)
+
         # Assume a default value of 30. Better for the sensor to operate with some
         # default value in this case rather than raise an exception.
         count = self._config['repository_sensor'].get('count', 30)
-
-        events = repository.get_events()[:count]
-        events = list(reversed(list(events)))
-
-        last_event_id = self._get_last_id(name=name)
-
+        events = repository.get_events()
+        event_list = []
         for event in events:
+            event_list.append(event)
+            if len(event_list) > count:
```

Comment:
I think this needs to be `==` and need to go before the append.  Otherwise the list will always have 1 more element than what we want (even when `count = 0` the list will have 1 element)

Reply:
Thanks!  I fix it.